trestle/tasks/xlsx_helper.py

```diff
@@ -405,3 +444,5 @@ def report_issues(self) -> None:
             logger.info(f'rows missing parameters: {self.rows_missing_parameters}')
         if len(self.rows_missing_parameters_values) > 0:
             logger.info(f'rows missing parameters values: {self.rows_missing_parameters_values}')
+        if len(self.rows_filtered) > 0:
+            logger.info(f'rows filtered: {self.rows_filtered}')
```

Comment:
can just be 
if self.rows_filtered:

Reply:
I have a lot of these in the xlsx_ code: if len(x) > 0. All fixed.